### PR TITLE
fix(curriculum): replace data urls with CDN urls for D3 projects

### LINKED ISSUE #401: need to spin off counties.json to freeCodeCamp/cdn
#### Browser Information

* Browser Name, Version: 
* Operating System: 
* Mobile, Desktop, or Tablet: 

#### Your Code / Link to Your Pen

```
 
 
 
```

### LINKED ISSUE #17755: Use CDN for Data Visualization projects user stories urls
#### Tell us about your browser and operating system

* Browser Name: 
* Browser Version: 
* Operating System: 

## Choropleth Map
https://learn.freecodecamp.org/data-visualization/data-visualization-projects/visualize-data-with-a-choropleth-map/  
screen shot 2018-06-28 at 11 27 27 pm  
### Change to: 
- US Education Data: `https://cdn.freecodecamp.org/testable-projects-fcc/data/choropleth_map/for_user_education.json`  
- US County Data: `https://cdn.freecodecamp.org/testable-projects-fcc/data/choropleth_map/counties.json`  
- CDN link: `https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js`
    

## Treemap Diagram
https://learn.freecodecamp.org/data-visualization/data-visualization-projects/visualize-data-with-a-treemap-diagram/
screen shot 2018-06-28 at 11 25 28 pm
### Change to: 
- Kickstarter Pledges:  `https://cdn.freecodecamp.org/testable-projects-fcc/data/tree_map/kickstarter-funding-data.json`
- Movie Sales:  `https://cdn.freecodecamp.org/testable-projects-fcc/data/tree_map/movie-data.json`
- Video Game Sales: `https://cdn.freecodecamp.org/testable-projects-fcc/data/tree_map/video-game-sales-data.json`
- CDN link: `https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js`

### PR DESCRIPTION
**Pull request by tbushman**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

This is to replace the urls that testable-projects D3 choropleth and tree-map reference to use the corresponding fCC CDN urls in place.  
In reference to two issues. 
Closes #17755  
Closes https://github.com/freeCodeCamp/testable-projects-fcc/issues/401

### CODE REVIEW

**Review by raisedadead — APPROVED** [review]

**Review by raisedadead — CHANGES_REQUESTED** [review]
There is a instance of `https://raw.githubusercontent.com/no-stack-dub-sack/testable-projects-fcc/master/src/data/choropleth_map/counties.json` that's still to be updated?

File: curriculum/challenges/chinese/04-data-visualization/data-visualization-projects/visualize-data-with-a-choropleth-map.chinese.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ localeTitle: 使用等值线图可视化数据
...
...悬停在某个区域上，并查看带有相应<code>id=&quot;tooltip&quot;</code> ，其中显示有关该区域的更多信息。 <strong>用户故事＃11：</strong>我的工具提示应该具有与活动区域的<code>data-education</code>相对应的<code>data-education</code>属性。以下是完成此项目所需的数据集： <br><ul><li> <strong>美国教育数据：</strong> <code>https://cdn.freecodecamp.org/testable-projects-fcc/data/choropleth_map/for_user_education.json</code> <strong>：</strong> <code>https://cdn.freecodecamp.org/testable-projects-fcc/data/choropleth_map/for_user_education.json</code> </li><li> <strong>美国县数据：</strong> <code>https://raw.githubusercontent.com/no-stack-dub-sack/testable-projects-fcc/master/src/data/choropleth_map/counties.json</code> <strong>：</strong> <code>https://cdn.freecodecamp.org/testable-projects-fcc/data/choropleth_map/counties.json</code> </li></ul>您可以通过分叉<a href="https://codepen.io/freeCodeCamp/pen/MJjpwO" target="_blank">此CodePen笔</a>来构建项目。或者您可以使用此CDN链接在您喜欢的任何环境中运行测试： <code>https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js</code> ： <code>https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js</code>完成后，将URL提交给您的工作通过所有测试的项目。如果卡住，请记住使用<a href="https://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514" target="_blank">Read-Search-Ask</a>方法。 </section>
```
[the remark this answers is not in the document]
Comment (tbushman) [reply curriculum/challenges/chinese/04-data-visualization/data-visualization-projects/visualize-data-with-a-choropleth-map.chinese.md]: @raisedadead Wow, thank you... bbiab

**Comment by tbushman** [comment]
@raisedadead Thank you for catching that. :D

**Comment by raisedadead** [comment]
Awesome! Thanks @tbushman

**Comment by tbushman** [comment]
OK @raisedadead Now I've caused CORS errors. I'm updating the projects on testable-projects and I have Access-Control cross-origin blocking errors from this source. Sorry, but I think it somehow needs to be handled on the server, because the 'jsonp', 'no-cors', and other client-side fetch headers don't fix.

**Comment by raisedadead** [comment]
Can you open up a fresh issue? Additional screenshots, GIF recordings, logs would be useful as well. Thanks.

**Pull request merged**